### DISCUSSION TITLE
Is there any howto for using kata in k8s managed envs? (EKS,GKE or AKS)

### DISCUSSION THREAD

**Discussion by aledbf** [kata-containers/kata-containers#5117] [Q&A] [answered]
Before raising this question

Have you looked at our [document](https://github.com/kata-containers/kata-containers/tree/main/docs) and still haven't found the answer?

Yes

Not sure this is supported or even possible.

I found a repository for EKS, but it's a bit old https://github.com/errordeveloper/kata-eks
Any pointer is welcome
Thanks

**Comment by GabyCT** [comment, marked as answer]
we do not have official documentation about this support, as we have not test it yet.
